lib/html_proofer/configuration.rb

```diff
@@ -86,6 +88,24 @@ def parse_json_option(option_name, config, symbolize_names: true)
           raise ArgumentError, "Option '#{option_name} did not contain valid JSON."
         end
       end
+
+      private
+
+      def default_options
+        PROOFER_DEFAULTS.merge(typhoeus: TYPHOEUS_DEFAULTS).merge(hydra: HYDRA_DEFAULTS).merge(parallel: PARALLEL_DEFAULTS)
```

Comment:
This does not feel like an elegant or bullet-proof way to enumerate all existing options to extract their default value type. Suggestions for how to improve this would be appreciated.